app/src/main/res/values/strings.xml

```diff
@@ -785,4 +784,7 @@ How to translate with transifex:
     <string name="nc_edit_icon">Edit Icon</string>
     <string name="get_invitations_error">Failed to fetch pending invitations</string>
     <string name="message_last_edited_by">Edited by %1$s</string>
+    <string name="share_link_to_conversation">Join conversation %1$s at %2$s</string>
+    <string name="note_to_self">Note to self</string>
```

Comment:
This won't work because translations for one language might differ (web vs app).

I saw this was also done wrong when "note to self" feature was implemented, see https://github.com/nextcloud/talk-android/pull/3351/files#diff-ae89ba240692c2742cc545fd2f57ae82d78e9fce03e3bedc5cfc1af8acbcc1a0R93

(This also causes https://github.com/nextcloud/talk-android/issues/3547)

I will fix this to not use translations at all for these checks and then adapt your PR.